README.md

```diff
@@ -1,119 +1,48 @@
 # Console Operator
 
-An operator for OpenShift Console.
+An operator for [OpenShift Console](https://github.com/openshift/console). The
+console-operator installs and maintains the web console on a cluster.
 
-The console-operator installs and maintains the web console on a cluster.
+## Workflow
 
-## Run on a 4.0.0 Cluster
+To quickly get started, check the [contrib](contrib/) and [examples](examples/)
+directories. For a more detailed overview, continue reading.
 
-The console operator is installed by default and will automatically maintain a console.  
+## Dependencies
 
-## Development Setup
+- Go 1.13 -- https://golang.org/dl/
 
-* Install Go 1.12 -- https://golang.org/dl/
-* GVM is recommended but not required -- https://github.com/moovweb/gvm
-
-## Clone the Repo & Build Locally
-
-To avoid some of the standard quirks of `gopath`, the recommended way to clone and 
```

Comment:
I removed this since we shouldn't need to backport to 4.2 once 4.5 ships, and 4.3 uses modules. It's no longer necessary.